src/Playground.res

```diff
@@ -1510,26 +1508,34 @@ let make = (~versions: array<string>) => {
   let subPanelRef = React.useRef(Nullable.null)
 
   let onResize = () => {
-    let newLayout = Webapi.Window.innerWidth < breakingPoint ? Column : Row
+    let newLayout = window.innerWidth < breakingPoint ? Column : Row
     setLayout(_ => newLayout)
     switch panelRef.current->Nullable.toOption {
     | Some(element) =>
```

Comment:
Here, `element` of type `WebAPI.DOMAPI.element` dont have the `style` property.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style

I could modify the height access the `height` property -> `element.style.height = "100px"`